CMSIS/DSP/Include/arm_math_types.h

```diff
@@ -98,6 +98,10 @@ extern "C"
 
 #if !defined(ARM_MATH_AUTOVECTORIZE)
 
+#if !defined(__ARM_FEATURE_MVE)
```

Comment:
Why is it needed here ?
If you build with Neon enabled, you have a test:

`#if __ARM_FEATURE_FP16_VECTOR_ARITHMETIC`

So you may have a similar problem ?

Perhaps it is better to use

`#if defined(__ARM_FEATURE_MVE)` 

before the tests for the \_\_ARM_FEATURE_MVE value.